projects/kerttu-global/src/app/kerttu-global-routing.module.ts

```diff
@@ -0,0 +1,67 @@
+/* tslint:disable:max-classes-per-file */
+import { Injectable, NgModule } from '@angular/core';
+import { PreloadingStrategy, Route, RouterModule, Routes } from '@angular/router';
+import { Observable, of as ObservableOf, timer as ObservableTimer } from 'rxjs';
+import { LocaleEnComponent } from '../../../laji/src/app/locale/locale-en.component';
+import { catchError, flatMap } from 'rxjs/operators';
+import { LocalizeGuard } from '../../../laji/src/app/locale/localize.guard';
+import { NotFoundComponent } from '../../../laji/src/app/shared/not-found/not-found.component';
+import { CheckLoginGuard } from '../../../laji/src/app/shared/guards/check-login.guard';
+import { OnlyLoggedIn } from 'projects/laji/src/app/shared/route/only-logged-in';
+
+@Injectable()
+export class PreloadSelectedModulesList implements PreloadingStrategy {
+  preload(route: Route, load: () => Observable<any>): Observable<any> {
+    const loadRoute = () => ObservableTimer(50).pipe(flatMap(() => load()), catchError(() => ObservableOf(null)));
```

Comment:
I did this the same way than in the `IUCN` project but it seems that it's a quite common practice to add some delay when preloading modules, to prevent network bottleneck

Reply:
Oh, I did some reading on this, neat!